src/common/librlist/rhwloc.c

```diff
@@ -15,6 +15,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <sys/utsname.h>
```

Comment:
some typo in commit message "missing to topo"?

Reply:
Maybe it needs a reword, but currently there's no typo (trying to keep the subject short probably makes it harder to read): "Add system info if missing to topo loaded from file"

I'll try to reword for clarity.